.bumpversion.cfg

```diff
@@ -0,0 +1,9 @@
+[bumpversion]
+current_version = 0.0.1
```

Comment:
Please start at `0.1.0`; we reserve that last number for actual patches (hot fixes)